packages/node_modules/@webex/internal-plugin-encryption/test/integration/spec/encryption.js

```diff
@@ -173,6 +173,38 @@ describe('Encryption', function () {
           .on('progress', spy))
         .then(() => assert.called(spy));
     });
+
+    it('checks body of the API call /downloads/endpoints', () => {
```

Comment:
If you could, write a test or two for `_fetchDownloadUrl()` to help us improve coverage for that method in the event that another contributor makes changes to this section of code.